app/Filters.scala

```diff
@@ -1,15 +0,0 @@
-import javax.inject.Inject
```

Comment:
this has moved to MyApplicationLoader `MyComponents.httpFilters` just below https://github.com/guardian/subscriptions-frontend/pull/1058/files#diff-751cafce06c7a60432eacdc822ece49dR74